src/api/explorerConfig/index.ts

```diff
@@ -189,6 +189,14 @@ const initialExplorerConfig: FullConfig = {
       version: "v3",
     },
   },
+  ethereum_goerli: {
+    id: "eth_goerli",
+    stable: {
+      base: "EXPLORER_STAGING",
```

Comment:
This is fine for testing a build, but we'll need a prod explorer before merging I guess.